share/spice/playing_cards/content.handlebars

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
I'm still not getting this to work. What's going wrong here? I feel like I'm missing something obvious.

```javascript
(function (env) {
    "use strict";
    env.ddg_spice_playing_cards = function(api_result){
        if(api_result.error || !api_result.cards) {
            return Spice.failed('playing_cards');
        }
        
        Spice.add({
            id: "playing_cards",
            name: "Deck of Cards",
            data: api_result.cards,
            meta: {
                sourceName: "DeckOfCards",
                sourceUrl: 'http://deckofcardsapi.com'
            },
            normalize: function(item) {
                return {
                    image: item.image
                }
            },
            templates: {
                group: 'movies',
                detail: false,
                variants: {
                    tile: 'poster'
                },
                options: {
                    content: Spice.playing_cards.content,
                    moreAt: false
                }
            }
        });
     }
  }
(this));
```